Vagrantfile

```diff
@@ -162,6 +175,11 @@ Vagrant.configure("2") do |config|
 	]
 	config.vm.provision :shell, :path => "puppet/preprovision.sh", :args => preprovision_args
 
+	# check if puppet is installed, if not, install it
```

Comment:
I could revert this PR that I did yesterday instead https://github.com/Chassis/Chassis/issues/937 Your PR installs Puppet 7.x and we're only on 5.5.10. I'd rather run the same version on both boxes as it'll help me with debugging as I don't have an M1 laptop.

Reply:
I actually don't know what version of puppet was used, so I defaulted to the latest, this was just me trying to fix puppet not being present, I'll defer to your judgement

Reply:
I've reverted that PR now so can you please have a crack at removing this and your script. Give it a test run and let me know if it works. If it does then I'm okay to merge it in. Could I be a pain and ask you to update the docs as well: https://github.com/Chassis/Chassis/tree/main/docs I'm happy for you to add in sections about M1 support with Parallels and some details about it. I'd do it myself but I don't know enough detail! Thanks!

Reply:
will do, though there's still a cavalcade issue here, switching back to puppet 5 may help with that. Once i have it operational then I'll look at the docs

Reply:
Awesome. Thanks again dude. Most appreciated seeing I don't have the hardware for this one!

Reply:
Alrighty. That Cavalcade issue should be [fixed](https://github.com/Chassis/Chassis/pull/944) if you `git pull` the latest version of Chassis.